pkg/server/spec/cloudsqlproxy_test.go

```diff
@@ -64,7 +66,58 @@ func TestNewCloudSQLProxyNil(t *testing.T) {
 	}
 }
 
+func TestNewCloudSQLProxyContainerFromEnvVar(t *testing.T) {
+	a := &app.App{
+		EnvVars: []*app.EnvVar{
+			&app.EnvVar{Key: "DB_PROJECT", Value: "project"},
+			&app.EnvVar{Key: "DB_ZONE", Value: "zone"},
+			&app.EnvVar{Key: "DB_NAME", Value: "name"},
+		},
+	}
+	csp := &CloudSQLProxy{
+		Instances:      "project:zone:name=tcp:3306",
```

Comment:
In order for this to work, it should be "", right? It only retrieves from env var when instances is "" (`if csp.Instances == "" {`) 

https://github.com/luizalabs/teresa/pull/588/files#diff-b43cdb99549d6afb049f5284a363b17bR68

Reply:
I think I'm actually testing the wrong place. 

`NewCloudSQLProxyContainer` does not create the proper string, it just supposes it's already created. I'll make more tests targetting `NewCloudSQLProxy`.